js/dataframe/shared/Table.svelte

```diff
@@ -549,7 +548,9 @@
 		col?: number,
 		dir?: SortDirection
 	): void {
-		const id = selected ? data[selected[0]][selected[1]]?.id : null;
+		const id = selected && selected[0] in data && selected[1] in data[selected[0]]
+		? data[selected[0]][selected[1]].id
+		: null;
```

Comment:
Can we rewrite this without the short circuiting + ternaries? I can't really make sense of this line.